tests/conftest.py

```diff
@@ -18,9 +18,8 @@
 
 
 @contextlib.contextmanager
-def cd_package(package):
+def cd_package(package_path):
```

Comment:
I would prefer to keep this fixture as-is. It seems you just need a chdir helper, like https://docs.python.org/3.11/library/contextlib.html#contextlib.chdir, so you can just extract that code and use the helper in this function.

Reply:
:+1: Done!